mobile/src/main/java/com/wedriveu/mobile/booking/view/BookingView.java

```diff
@@ -0,0 +1,14 @@
+package com.wedriveu.mobile.booking.view;
+
+import com.wedriveu.mobile.booking.presenter.model.BookingPresentationModel;
+
+/**
+ * Created by nicolalasagni on 28/07/2017.
```

Comment:
please insert proper javadoc comments here and in other interfaces. Remember you can use autogenerated layout for this.